src/interface.js

```diff
@@ -0,0 +1,47 @@
+$(document).ready(function (){
+  var game = new Game();
+
+  $( '#bowlingBall' ).click(function() {
```

Comment:
There's a 'smell' here.  The fact that you're calling several different methods that accept the `game` as a parameter, one after the other, makes me think that there's something which could be abstracted from here into another method.

Also, the interface seems to combine 'game engine' and presentation functionality.  Printing score is the latter.  Checking whether a roll is valid or the game is over is the former.  Interface sounds like a presentational layer so the engine-like bits feel out of place.  Perhaps some of them could be moved to the game?

Reply:
Could an abstraction be a method that calls all these methods that use game? This method would then have game flowing through all of them (like how my score calculator works)